lib/shopify-cli/tasks/create_api_client.rb

```diff
@@ -17,6 +17,15 @@ def call(ctx, org_id:, title:, type:)
           redir: [OAuth::REDIRECT_HOST]
         )
 
+        unless resp
+          ctx.abort("Error - empty response")
+        end
+
+        errors = resp.dig("errors")
```

Comment:
While working on this I found that some kinds of errrors returned by Partners were not being handled correctly.

Reply:
There are some edge cases where we get back the HTML error page from partners, I wonder if there is anything we can do about those.

Reply:
I think we can address that separately. There's already an assumption below that the response is a hash.